python/lsst/cp/pipe/defects.py

```diff
@@ -645,8 +646,9 @@ class MeasureDefectsCombinedTask(MeasureDefectsTask):
     _DefaultName = "cpDefectMeasureCombined"
 
 
-class MeasureDefectsCombinedWithFilterConnections(MeasureDefectsCombinedConnections,
-                                                  dimensions=("instrument", "detector")):
+class MeasureDefectsCombinedWithFilterConnections(pipeBase.PipelineTaskConnections,
+                                                  # MeasureDefectsCombinedConnections,
```

Comment:
What does this comment mean?

Reply:
This comment and the previous one removed, as we're avoiding potential connection inheritance issues.